raiden/storage/migrations/v21_to_v22.py

```diff
@@ -236,16 +236,16 @@ def _add_canonical_identifier_to_statechanges(
             )
 
             if should_delete:
-                delete_state_changes.append(state_change_record.identifier)
+                delete_state_changes.append(state_change_record.state_change_identifier)
```

Comment:
yeah, good find! I have to admit, I get confused by the naming in https://github.com/raiden-network/raiden/blob/master/raiden/storage/sqlite.py#L13-L27

imho, we should stick to `.identifier` for the unique id field and only use the qualifiers for the cross referenced ids (i.e. `EventRecord.state_change_identifier`)

Reply:
btw, this is not for this very PR, just a drive-by. I'll resolve and create an issue instead.

Reply:
Yes, that sounds like a good idea.